Source/CMakeLists.txt

```diff
@@ -180,7 +180,8 @@ target_compile_definitions(woof PRIVATE MINIZ_NO_ARCHIVE_APIS)
 
 # Assemble library files.
 set(WOOF_DLLS "")
-if(WIN32 AND NOT MINGW)
+# If SDL2_DIR is set, libraries are downloaded from libsdl.org
+if(WIN32 AND SDL2_DIR)
```

Comment:
We do not yet prevent mixing of downlaoded and native libraries, right?

Reply:
We do if there are no old lib files in target dir.

Reply:
What I mean is, we do not enforce the "if one library is downloaded, then all libraries are downloaded" rule.

Reply:
Yes, if SDL2_DIR is set, then all libraries are downloaded.

Reply:
Sorry, I fail to see this among the changes of this PR. 🤷‍♂️

Reply:
Ah, you mean because it is impossible to have the SDL2_Mixer or SDL2_Net packages installed without also having the SDL2 package installed - as per package dependencies.

Reply:
But in theory, users can install SDL2 from MSYS2, but not SDL2_mixer and SDL2_net. Then yes, libs are mixed. Maybe we should just don't download libsdl.org packages on MSYS2.

Reply:
Or just download of SDL2_DIR is set.

Reply:
However, there is a problem with the target bin directory. If users download the libsdl.org libraries on MSYS2, they cannot install them, so we have to copy the lib files to the target directory.
I think we are too helpful, this is for the developers. Just do pacman -S.

Reply:
1. We may assume a clean build directory.
2. I would like to replace most of our checking for build system and compiler with a simple "libraries were downloaded or not" logic, because this is what it boils down to in most cases.